src/CMakeLists.txt

```
@@ -32,6 +32,7 @@ if(OTBR_BORDER_AGENT)
 endif()
 add_subdirectory(common)
 add_subdirectory(ncp)
+add_subdirectory(nat64)
```

```suggestion
add_subdirectory(tayga_nat64)
```

I think this directory is dedicated for tayga?

---

Tayga is one kind of implementation for NAT64. Other implementation can also be added into the same directory. If size of the directory becomes a concern later, we can refactor the structure accordingly.

---

SGTM.